src/spyglass/sharing/sharing_kachery.py

```diff
@@ -189,20 +133,23 @@ def make(self, key):
         # note that we're assuming that the user has initialized a kachery-cloud client with kachery-cloud-init
         linked_key = copy.deepcopy(key)
```

Comment:
Linked key doesn't get used?

```suggestion
```

Reply:
Very true; it's in there because eventually we might want to be able to share linked files as well, so it's a place holder, but I'll note that and comment it out.